CONTRIBUTING.md

```diff
@@ -317,6 +317,8 @@ GitHub provides reactions as a simple and quick way to provide feedback to pull
 
 Pull request reviews should include a list of what has been reviewed in a comment, so other reviewers and mergers can know the state of the review.
 
+Pull request reviews that only nitpick things that are easily fixed in treewides cause unnecessary friction and are better left unposted. Nitpicks may however accompany blockers.
```

Comment:
> Nitpicks may however accompany blockers.

This creates the weird incentive to find a blocker to justify many nitpicks.

I like [C4's guideline on this](https://rfc.zeromq.org/spec/42/#24-development-process):

> 14\. Maintainers SHALL NOT make value judgments on correct patches.
> 
> 18\. Any Contributor who has value judgments on a patch SHOULD express these via their own patches.

In general I'd like to move more towards C4, which is well elaborated [here](https://hintjens.gitbooks.io/social-architecture/content/chapter4.html) (highly recommended reading).

Reply:
This is a great resource, i'll add it to my reading.

> This creates the weird incentive to find a blocker to justify many nitpicks.

Weird incentive to improve the review quality?

Reply:
I have some thoughts about committer vs. contributor time and activation energy and the context switch overhead of reviewing vs. patching here that make me worry that the expectation to provide all not‐absolutely‐critical feedback as separate PRs from the reviewer will be a net drag on maintainer resources over time, but they’re not fully‐formed enough for me to express them other than by gesturing broadly at them here. (This is, admittedly, probably partly a workflow problem. But we’re stuck on GitHub for now…)